src/DynamoPackages/Package.cs

```diff
@@ -572,21 +575,66 @@ internal void SetAsLoaded()
             RaisePropertyChanged(nameof(LoadState));
         }
 
+        AssemblyLoadContext GetPackageALC()
+        {
+            if (LoadedAssemblies.Any())
+            {
+                if (LoadedAssemblies.All(x => AssemblyLoadContext.GetLoadContext(x.Assembly) != AssemblyLoadContext.Default))
+                {
+                    return AssemblyLoadContext.GetLoadContext(LoadedAssemblies[0].Assembly);
+                }
+            }
+            return AssemblyLoadContext.Default;
+        }
+
+
+
         internal void UninstallCore(CustomNodeManager customNodeManager, PackageLoader packageLoader, IPreferences prefs)
         {
             if (LoadedAssemblies.Any())
             {
+                bool unloadAnyPackage = DynamoModel.FeatureFlags?.CheckFeatureFlag("UnloadAnyIsolatedPackage", false) ?? false;
```

Comment:
can you describe what these two flags are meant for?

Reply:
UnloadAnyIsolatedPackage - activate the ALC unload feature for all packages.
THe other one is for activating the ALC unload feature for a specific package (identified by name)